source/includes/api/_user_management.md

```diff
@@ -275,3 +275,18 @@ curl -X DELETE http://api.resourcewatch.org/auth/user/<user_id>
     ...
 }'
 ```
+
+## Permissions
```

Comment:
Can you move this elsewhere? in all other API endpoints, permissions are described per endpoint, not in a separate section. I don't see a good reason to break that consistency